### PR TITLE
Service rifle and trooper armor adjustments

### PR DESCRIPTION
**Pull request by Osmos123**
## About The Pull Request

Aims to make the trooper role more robust considering many new players and role players choose this role.

Removes the -10 bullet penalty to the vest and instead replaces with -10 laser to differentiate from corporal armor. (The armor is still inferior to vet legion armor and slightly better than prime armor)

The more controversial change is with the service rifle. In my personal opinion it is a trap weapon. You try fighting legion with this who will most likely meta with 10mm/grease guns you will get destroyed. The guns fire rate is ugly to use and is something you will actively want to replace from a tdm standpoint. I don't remember the service rifle being this bad in the previous servers. The changes make it essentially a scopeless marksmen carbine in terms of performance with more movement slowdown.

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
tweak: Buffs the service rifle (more movement slowdown in exchange for performance similar to the marksman carbine)
balance: Removal of the bullet penalty for the trooper armor and adds laser penalty to compensate.
/:cl:

### CODE REVIEW

**Comment by RicktenStone** [comment]
The service rifle buff is fine (although making it 'too' strong is always something that should be careful to tread around, since its a gun thats available for even the lowest rank in the ncr and is solid to the average weapon of the legion which are lever action repeaters), and one has to be clear that your example of 10mm smgs and grease guns are flawed, as only veterans (Who are ranger equal elite roles) and assassins spawn with 10mm smgs, and grease guns are not a legion spawn, thus comparing them to service rifles is inaccurate and a bad example. A better example is comparing them to the common legion weapons which are cowboy repeaters/hunting shotguns/hunting rifles that the primes/recruits/explorers use, not what their elites use. Either way, sure buff the service rifle.

What is not fine, however, is buffing the ncr trooper armor to be better than the prime armor. This would make even the lower echelons of the ncr hierarchy have better armor then both the lower and mid echelons of the legion hierarchy, with only their elite roles (veterans) having better armor than an ncr grunt. So not only will they have an automatic gun that is now boosted to fire at a much faster rate compared to the lever action weapons of most legion roles, but they will also be out-armoring all of the legion roles besides the legion elites, which is kind of insane for a literally for a ground-floor tier grunt role that makes up the majority of the ncr faction.

Again, the flaws in the argument are 
a-) 10mm smgs are an elite loadout spawn for legion (Veterans, assassin) and are not comparable to the service rifles, they can only be compared to NCR elite loadout guns (Ranger r91, Sergeant's ak) in which they are actually outclassed, instead the service rifle should be compared to the cowboy repeater
b-) NCR trooper armor should not be compared to the legion veteran armors or be close to them in stats (Legion veteran is an elite role, ncr trooper is the entry level grunt role), at most they should be equal to legion prime armors (And even then, pretty sure there are less legion prime slots than there are NCR slots

tldr, buff the service rifle but not the trooper armor, go find combat armor like everyone else

**Comment by DorfGuy1** [comment]
yeah the service rifle, was just disspointing. I'll leave the rest for people to hash out

**Comment by Osmos123** [comment]
Upon rethinking, buffing a single rifle does little. In my opinion all manner of automatics should be nerfed across the board. I don't think I've ever seen a ranger use the trail carbine loadout over the r91 or any of the vet legion using the lever-action shotguns over the 10mm smg cause automatics are that great. The extra damage does not make up for the lower ammo capacity and having to press the activate button on each shot (cancer on 50% td).
